src/builder/MorphoInfo.sol

```diff
@@ -0,0 +1,423 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity ^0.8.23;
+
+import {MarketParams} from "src/interfaces/IMorpho.sol";
+import {HashMap} from "./HashMap.sol";
+
+library MorphoInfo {
+    error UnsupportedChainId();
+    error MorphoMarketNotFound();
+    error MorphoVaultNotFound();
+
+    // Note: Current Morpho has same address across mainnet and base
+    function getMorphoAddress() internal pure returns (address) {
+        return 0xBBBBBbbBBb9cC5e90e3b3Af64bdAF62C37EEFFCb;
```

Comment:
Ideally, this function takes a chain id and reverts if it's an unsupported chain.

Reply:
Sounds good to me will address this.